README.md

```diff
@@ -40,7 +43,7 @@ To build the code without running the tests, add to the "clean build" command th
 ```
 
 ### Testing
-To run *all* tests:   
+To run *all* tests:
```

Comment:
The README seems to be missing a dedicated test command to run docker tests only, as the other package managers have.